docs/_docs/reference/experimental/named-tuples.md

```diff
@@ -100,6 +100,24 @@ The translation of named tuples to instances of `NamedTuple` is fixed by the spe
  - All tuple operations also work with named tuples "out of the box".
  - Macro libraries can rely on this expansion.
 
+### The FieldsOf Type
```

Comment:
Maybe it should be `NamedTuple.Of[T]` or `NamedTuple.From[T]`?

Reply:
`From` looks like it would work nicely!

Reply:
Great. In this case, I think the usecase would always be to reference the whole `NamedTuple.From` instead importing `From` and using just that. It gives a clearer understanding of what is expected.